src-java/kilda-model/src/main/java/org/openkilda/model/Flow.java

```diff
@@ -106,7 +106,7 @@ public Flow(@NonNull String flowId, @NonNull Switch srcSwitch, @NonNull Switch d
                 .priority(priority).pinned(pinned).pathComputationStrategy(pathComputationStrategy)
                 .targetPathComputationStrategy(targetPathComputationStrategy)
                 .loopSwitchId(loopSwitchId).affinityGroupId(affinityGroupId)
-                .yFlowId(yFlowId);
+                .yFlowId(yFlowId).yFlow(yFlow);
```

Comment:
What is the purpose to include YFlow entity in equal and hashcode calculations? yFlowId is enough.

Reply:
YFlow entry has been removed from the equal and hashcode calculations.